src/utils/offers/offerAggregateOptions.js

```
@@ -50,11 +50,15 @@ const offerAggregateOptions = (query, params) => {
   }
 
   if (proficiencyLevel) {
-    match.proficiencyLevel = { $in: proficiencyLevel }
+    const decodedValue = decodeURIComponent(proficiencyLevel).split(',')
+
+    match.proficiencyLevel = { $in: decodedValue }
   }
 
   if (price) {
-    const [minPrice, maxPrice] = price
+    const decodedPrice = decodeURIComponent(price).split(',')
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)